.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
Try to only commit files related to the given problem in this branch. Don't worry about fixing this, but note for the future.

Also I would add `.idea` to my Git ignore file. These files change constantly and are just related to editor configuration.